changelogs/CHANGELOG-v0.5.1.md

```diff
@@ -0,0 +1,36 @@
+The v0.5.1 release is a patch release which introduced bytecode image spec update to support multi arch and
+update XDP dispatcher to point to a released tag
```

Comment:
```suggestion
The v0.5.1 release is a patch release that introduced a bytecode image spec update to support multiple images and adds build tooling for bytecode images.  It also makes some changes so that the main branch uses the "latest" versions of the XDP and TC dispatchers, and released versions use the version supported by the release.
```